### ISSUE TITLE
CommandBar redundantly sets a default aria-label on items when ariaLabel is not specified

### ISSUE THREAD

**Issue by trevorsg** [OfficeDev/office-ui-fabric-react#2266]
Fix coming soon.

Repro: Create a command bar with a command called "Foo" with no ariaLabel property. 
Expected: No aria-label
Actual: The aria-label and the button text are the same, which is redundant (Keros violation).

**Comment by trevorsg** [comment]
From http://aka.ms/keros-redundant-labels-instructions: 
- If a native widget has has multiple instructions and cues, they must not be redundant.
- Consider all of the labels and instructions that you have identified for this widget.
Determine whether any are redundant. (They can be redundant without being identical.)
If the widget has only a single label, choose N/A.
If the labels and instructions include redundant information, choose **Fail**.
If the they do not include any redundant information, choose Pass.
